plotter/static/chromDisplay.js

```diff
@@ -527,8 +528,8 @@ function chromDisplay(svg, meta, plotter) {
                 var url = window.location.href;
                 var res = url.indexOf("profile_old");
 
-                if (res == -1)
-                    width = 1250;
+                // if (res == -1)
+                //     width = 1250;
```

Comment:
Before I commented this out, the width of the SVG in the old image viewer was not working correctly for large zoom levels (ipad and above were having a width of 1250 but they are actually much bigger than that)